tests/acceptance/features/apiSpaces/spaceManagement.feature

```diff
@@ -0,0 +1,171 @@
+@api @skipOnOcV10
```

Comment:
I thought that we already deleted all the `skipOnOcV10` tags.
I will do that as a separate PR.